.github/workflows/deploy_CI.yml

```diff
@@ -34,18 +34,18 @@ jobs:
         if: ${{ github.event.ref }} =~ ^refs/tags/v[0-9]+.[0-9]+.[0-9]
         run: echo "TAG=$(echo $GITHUB_REF | cut -d / -f 3)" >> $GITHUB_ENV
 
-      - name: Set env for staging or dev
-        if: endsWith(github.ref, '/develop') || contains(env.TAG, '-beta')
+      - name: Set env for production if master branch
+        if: endsWith(github.ref, '/master')
         run: |
-          echo "ENVIRONMENT=${{ secrets.DEV_ENV }}" >> $GITHUB_ENV
-          echo "HOST=${{ secrets.DEV_HOST }}" >> $GITHUB_ENV
+          echo "ENVIRONMENT=${{ secrets.PROD_ENV }}" >> $GITHUB_ENV
+          echo "HOST=${{ secrets.PROD_HOST }}" >> $GITHUB_ENV
           echo "USERNAME=${{ secrets.USERNAME }}" >> $GITHUB_ENV
-          echo "WWW_SITE_URL=dev.devinit.org" >> $GITHUB_ENV
-          echo "SITE_URL=dev.devinit.org" >> $GITHUB_ENV
-          echo "DATA_SITE_URL=dev.devinit.org" >> $GITHUB_ENV
+          echo "WWW_SITE_URL=www.devinit.org" >> $GITHUB_ENV
+          echo "SITE_URL=devinit.org" >> $GITHUB_ENV
+          echo "DATA_SITE_URL=data.devinit.org" >> $GITHUB_ENV
 
-      - name: Set env for production
-        if: endsWith(github.ref, '/master') || contains(env.TAG, '-beta') != true
+      - name: Set env for production if release and not beta
+        if: startsWith(github.ref, 'refs/tags/v') && ( contains(env.TAG, '-beta') != true )
```

Comment:
I think the checks on line 38 should be the same as on this line... 
Also propose to remove line 7, so we only deploy to production by way of tags strictly, and not pushing to master. That should go a long way in reducing the confusion (or complexity) of this

Reply:
Okay, I removed line 7 and removed the whole deploy to live on master branch ref section.